README.md

```diff
@@ -23,41 +23,69 @@ The server component to the TourTrak system built using Django.
 * rabbitmq-server
 
 
-## Install PostGIS
+## Install Postgres with PostGIS (Install The Database)
 
-1. Install postgreSQL. Refer to official documentation. After, install postGIS, which is a postgreSQL extention for handling spatial data by doing `apt-get install postgresql-9.1-postgis`
-2. Setup the postgreSQL database:
+<i>(This is where you begin the server setup instructions)</i>
```

Comment:
(a) `*content*`
(b) if we have to call out that the 'Install' section installs things, we are probably doing it wrong